src/analysis/mdFBA/buildMDFBAProblem.m

```diff
@@ -78,8 +78,14 @@
 
 %Osense and csense might be missing, if osense is missing, we assume
 %maximisation.
-if isfield(model,'osense')
-    osense = model.osense;
+if isfield(model,'osenseStr')
+    if strcmpi(model.osenseStr,'min') 
```

Comment:
Would it make sense to put these few lines in a separate function, e.g., `setOsense(model)`?